site/reference/fn/render/README.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "`render`"
+linkTitle: "render"
+type: docs
+description: >
+   Render a package
+---
+<!--mdtogo:Short
+   Render a package.
+-->
+
+`render` executes the pipeline of functions defined in the Kptfile [kptfile reference].
```

Comment:
`render` does more than just execute functions. It also:
1) Preconditions: Validates meta resources (i.e `Kptfile` and `functoinConfig`)
2) Postconditions: Formats the packages

Reply:
I think `validating meta resources` is expected by default, right ? Do we need to mention is explicitly ? 
My line of thinking: Since `Kptfile` and `functionConfig` are input to the pipeline and `render` is in business of providing that input to the pipeline, it should be validating those.